### PR TITLE
Suggest a quicker and easier installation method

### PR DESCRIPTION
**Pull request by sanmai-NL**
This is particularly useful to keep CI pipelines fast and concise.

### CODE REVIEW

**Comment by sanmai-NL** [comment]
@badboy This seems like an easy to review PR after your holidays.

File: README.md [hunk 1]
```diff
@@ -26,19 +26,39 @@ in your book.
...
+```
+  yes | cargo binstall mdbook-mermaid
+ INFO resolve: Resolving package: 'mdbook-mermaid'
+ WARN The package mdbook-mermaid v0.12.6 will be downloaded from github.com
+ INFO This will install the following binaries:
+ INFO   - mdbook-mermaid (mdbook-mermaid -> /home/user/.cargo/bin/mdbook-mermaid)
+Do you wish to continue? yes/[no]
+?  INFO Installing binaries...
+ INFO Done in 1.248059328s
+```
```
Comment (badboy) [inline README.md:45]: No need for the full output, the command is enough.

File: README.md [hunk 2]
```diff
@@ -26,19 +26,39 @@ in your book.
 
 ## Installation
 
-If you want to use only this preprocessor, install the tool:
+### Install `mdbook-mermaid`
+
+If you want to use only this preprocessor, install the tool.
+
+The simplest and quickest way to install is by [Cargo `binstall`](https://github.com/cargo-bins/cargo-binstall). 
+This installs the right binaries released here on GitHub (which you could also download manually).
```
Comment (badboy) [inline README.md:34]: Indeed they are and we should call that out!

**Review by badboy — CHANGES_REQUESTED** [review]
Appreciate the work and I will add that install method, but I'm quickly wipping up the commit myself now.

**Pull request closed without merging**